Resources/Audio/Ambience/Antag/attributions.yml

```diff
@@ -1,12 +1,12 @@
-- files: ["﻿nukeops_start.ogg"]
+- files: ["nukeops_start.ogg"]
   license: "CC-BY-NC-SA-3.0"
-  copyright: "Taken from TG station."
-  source: "https://github.com/tgstation/tgstation/commit/827967c9650c23af64280ad1491405fed8f644c5"
+  copyright: "Taken from TG station"
+  source: "https://github.com/tgstation/tgstation/commit/827967c9650c23af64280ad1491405fed8f644c5#diff-6cc910b7cad9ac4333c8d0885fc844746066120b465d8d4ba8f7019169316574"
```

Comment:
Sound isn't TG the PR from last week made a mistake.

Reply:
What you mean? The sounds are the same and it isnt mentioned in their attributions or license files. So it has to be from tgstation.

Reply:
It's not NC

Reply:
Oh you mean the license is wrong? Alr I will fix when I get back from school.